File: 01142/src/main/java/EA6/Konvergenzrate.java
package EA6;

public class Konvergenzrate {

	private static double a(int n) {
		return 1.0/(Math.pow(2.0, Math.pow(2, Math.pow(2, n))));
	}
	
	private static double a = 0.0; // Grenzwert


	public static void main(String[] args) {
		for (int i=0; i<10; i++) {
			for (int j=0; j<5; j++) {
				System.out.println((a(j+1)-a)/Math.pow((a(j)-a),i));
			}
		}
	}
}

File: 01142/src/main/java/EA6/Konvergenzrate.kt
package ea6

import kotlin.math.pow

private fun a(n: Int): Double = 1.0 / 2.0.pow(2.0.pow(2.0.pow(n.toDouble())))

private const val a = 0.0 // Grenzwert

fun main() {
    (0..9).forEach { i ->
        (0..4).forEach { j ->
            println((a(j + 1) - a) / (a(j) - a).pow(i.toDouble()))
        }
    }
}
